setup.py

```diff
@@ -27,6 +27,10 @@
         "django-parler<2.0",
         "django-anymail",
         "django-mailer",
+        "graphene>=2.0",
```

Comment:
I'm wondering if we want to force graphene/graphql as a requirement or would using the `extras_require` feature be a good way to go here? Not sure what other changes that might require for the implementation. :shrug: [[Declaring “Extras” (optional features with their own dependencies](https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies)] Also not sure if this optional requirements thing is a good way to go, but just thinking out loud.

Reply:
The `extras` sound like the reasonable approach, considering that
> Sometimes a project has “recommended” dependencies, that are not required for all uses of the project

which seems to be the exact case. I think as long as we document clearly on the README that there's the API and what it requires to use it (i.e. installing the extra deps), we should be good.

Reply:
Alex has now implemented graphql support as `extras` 👍